ereol_frontend/ereol_frontend.pages_default.inc

```diff
@@ -159,6 +159,14 @@ search/ting
 <none>
 ',
     'name' => 'ereolen-search',
+    'metatag_panels' => array(
+      'enabled' => 1,
+      'metatags' => array(
+        'title' => array(
+          'value' => 'Søgning | %keywords:html_safe | [current-page:pager][site:name]',
```

Comment:
You should use the the t() function and keep the english strings

Reply:
Of course! I've fixed this and then we have to fix our translation which are off.